services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell.pipeline.service
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectId, UserId, Version }
+import org.mockito.Mockito.when
+import org.mockito.{ Matchers => MockitoMatchers }
+import org.scalatest.{ AsyncWordSpec, BeforeAndAfterAll, Matchers }
+import org.scalatestplus.mockito.MockitoSugar
+import java.net.URLEncoder
+import java.nio.file.Paths
+import scala.concurrent.{ ExecutionContext, Future }
+
+class GitLabProjectVersioningTest extends AsyncWordSpec with Matchers with MockitoSugar with BeforeAndAfterAll {
```

Comment:
As I can see, you don't need BeforeAndAfterAll here

Reply:
You got me! Thanks!